components/questions/question-types/BinaryQuestion.tsx

```diff
@@ -101,6 +101,9 @@ export function BinaryQuestion({
                 )}
                 onKeyDown={onDateKeydown}
                 onMouseDown={(e) => e.stopPropagation()}
+                onChange={(e) => {
+                  setValue("resolveBy", e.currentTarget.value)
+                }}
                 ref={(e) => {
                   resolveByInputRef.current = e
                   register("resolveBy", { required: true }).ref(e)
```

Comment:
This `register` call needs to be pulled out to parent scope and unpacked into the props here like ` {...register("sharePublicly")}`, I think